src/main/apps/dataMigration/migrateSourcesJson.ts

```diff
@@ -36,5 +37,5 @@ export const migrateSourcesJson = () => {
         })
     );
 
-    writeSourcesFile(migratedSources);
+    writeV1SourcesFile(migratedSources);
```

Comment:
I'm sure it's not wrong but I can't understand the V1 still being written. It is definitely due to a lack of general infrastructure understanding of the sources part

Reply:
I think it is simpler this way. 

We already have the migration `sources.json` → `sources-versioned.json`.v1

Now I only had to write `sources-versioned.json`.v1 → `sources-versioned.json`.v2

If I wouldn't write v1, I would have to write two migrations:
- `sources.json` → `sources-versioned.json`.v2
- `sources-versioned.json`.v1 → `sources-versioned.json`.v2

I could implement the first by doing `sources.json` → `sources-versioned.json`.v1 in memory and then doing the second migration. But still this would mean that the second migration can work on either content from memory of from disk. 

Currently this implementation seems simpler to me, as I can mostly leave the existing code. But I also didn't try another implementation, so maybe my speculations are wrong. :-)